code/go/internal/validator/spec.go

```diff
@@ -61,16 +61,17 @@ func (s Spec) ValidatePackage(pkg Package) ve.ValidationErrors {
 	}
 
 	// Syntactic validations
-	errs.Append(rootSpec.validate(pkg.Name, pkg.RootPath))
+	errs = rootSpec.validate(pkg.Name, pkg.RootPath)
+	if len(errs) != 0 {
+		return errs
+	}
```

Comment:
With this change, do we even need the rest of the changes in this PR? I would assume the lack of a `version` value in `manifest.yml` would get caught at this point and we would never get to the semantic validations?

Reply:
Yes, but on the other hand that code is unsafe - lack of type check. I would be for leaving it as is (less error prone code).

Reply:
Fair point. Let's keep the type assertion.